code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm

```diff
@@ -472,8 +472,8 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
 	var/obj/effect/temp_visual/banishment_portal/portal = null
 	///The timer ID of any Banish currently active
 	var/banish_duration_timer_id
-	///Luminosity of the banished target
-	var/stored_luminosity
+	///Phantom zone reserved area
+	var/datum/turf_reservation/reserved_area
```

Comment:
This should probably be one per ability but I'm unsure about how bad it is to spam reserve and unreserve so I'm going to @spookydonut

Reply:
reserve once and dont release it, reuse.